living_documentation_generator/utils/github_rate_limiter.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import time
+from datetime import datetime
+from typing import Optional, Callable
+from github import Github
+
+
+class GithubRateLimiter:
+    def __init__(self, github_client: Github):
+        self.github_client = github_client
+
+    def __call__(self, method: Callable) -> Callable:
+        def wrapped_method(*args, **kwargs) -> Optional:
```

Comment:
Optional should be usually paired with other type.

Reply:
Yeah, I have update it to Optional[Any].

Reply:
Please make it a `Any | None`. Optional type hint is outdated.

Reply:
Fixed in commit a7771c6b6a6963669fdcd12803ce44009e8d7880.